5.0/en/0x03-Using-ASVS.md

```diff
@@ -102,13 +100,13 @@ Level 1 is not necessarily penetration testable by an external tester without in
 
 ### Level 2
 
-Most applications should be striving to achieve this level of security. It contains around 50% of the requirements in the ASVS.
+Most applications should be striving to achieve this level of security. Around 50% of the requirements in the ASVS are L2 meaning that an application needs to implement around 70% of the requirements in the ASVS in order to comply with L2.
```

Comment:
Let's make it clear that 70% means including Level 1 requirements

Reply:
ok